app/controllers/messages_controller.rb

```diff
@@ -5,16 +5,22 @@ class MessagesController < ApplicationController
 
   def index
     blocked_ids = Block.where(blocking_user: current_user).pluck(:blocked_user_id)
+
+    includes = []
     if params[:view] == 'outbox'
       @page_title = 'Outbox'
       from_table = current_user.sent_messages.where(visible_outbox: true).ordered_by_thread.select('distinct on (thread_id) messages.*')
       from_table = from_table.where.not(recipient_id: blocked_ids).joins(:recipient).where.not(users: { deleted: true })
+      includes << :recipient
     else
       @page_title = 'Inbox'
       from_table = current_user.messages.where(visible_inbox: true).ordered_by_thread.select('distinct on (thread_id) messages.*')
       from_table = from_table.where.not(sender_id: blocked_ids).left_outer_joins(:sender).where('users.deleted IS NULL OR users.deleted = false')
+      includes << :sender
```

Comment:
not sure why you're doing this as array appending rather than a ternary after this bit, but w/e, that's not important

Reply:
Oh, I was originally using `:first_thread` - I would've also just done a variable here like `include` but that's a keyword so I didn't fix it up.

I could use `inc = :recipient` and `inc = :sender`